src/machines/modelingMachine.ts

```diff
@@ -2033,12 +2040,6 @@ export const modelingMachine = setup({
           }
         }
 
-        const valueOrVariable = (variable: KclCommandValue) => {
```

Comment:
Ohh this is where this was, nice idea moving it out to utility

Reply:
Yes!

Reply:
So re: https://github.com/KittyCAD/modeling-app/pull/6167/files#r2049136972

This is why I quickly moved it there so I would re-implement it twice. But yeah could use a better name and a comment